syntax = "proto3";

package flow;

message ChannelConfig {
  string name = 1;
  string address = 2;
  int64 timeout = 3;
  map<string, int64> method_timeout = 4;
}

message GrpcClientConfig {
  repeated ChannelConfig channel_configs = 1;
}


